src/Raven.Server/Documents/ETL/Providers/AI/Embeddings/EmbeddingsGenerationRetryStrategy.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Raven.Server.Documents.ETL.Providers.AI.Embeddings;
+
+public enum EmbeddingsGenerationRetryStrategy
```

Comment:
Is there a reason to _have_ this strategy? We never offer this in any other place, and more configuration is usually bad for us.